src/services/room-service.spec.js

```diff
@@ -4,8 +4,10 @@ import RoomService from './room-service.js';
 import uniqueId from '../utils/unique-id.js';
 import Database from '../stores/database.js';
 import RoomStore from '../stores/room-store.js';
+import RoomLockStore from '../stores/room-lock-store.js';
 import { ROOM_ACCESS_LEVEL } from '../domain/constants.js';
 import { destroyTestEnvironment, setupTestEnvironment, pruneTestEnvironment, setupTestUser, createTestRoom } from '../test-helper.js';
+import { afterEach } from '@jest/globals';
```

Comment:
Not needed

Reply:
Autoimport, I didn't see it sorry, I still haven't found a way to disable this one

Reply:
Fixed